src/styles/index.scss

```diff
@@ -1,3 +1,229 @@
 body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   margin: 0;
 }
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+  padding: 10px;
+  display: flex;
+  position: absolute;
```

Comment:
Using 'position: absolute;' on the '.calendar' class might cause layout issues if the parent element doesn't have a defined position. Ensure that the parent element is positioned correctly to avoid unexpected behavior.